ehrapy/preprocessing/encoding/_encode.py

```diff
@@ -832,44 +843,46 @@ def _get_categoricals_old_indices(old_var_names: list[str], encoded_categories:
     return idx_list
 
 
-def _add_categoricals_to_obs(ann_data: AnnData, categorical_names: list[str]) -> None:
+def _add_categoricals_to_obs(from_: AnnData, to: AnnData, categorical_names: list[str]) -> None:
```

Comment:
Why does from have an underscore at the end and to does not?
The naming is a little bit weird...Can we make this more explicit?

Reply:
from is a reserved keyword in python so the underscore was needed. Maybe add it to `to` as well?